src/prompts/text.rs

```diff
@@ -262,12 +274,11 @@ impl<'a> From<Text<'a>> for TextPrompt<'a> {
             default: so.default,
             help_message: so.help_message,
             formatter: so.formatter,
-            validators: so.validators,
             suggester: so.suggester,
             input: so
                 .placeholder
-                .map(|p| Input::new().with_placeholder(p))
-                .unwrap_or_else(Input::new),
+                .map(|p| Input::new_with(so.text.unwrap_or("")).with_placeholder(p))
+                .unwrap_or_else(|| Input::new_with(so.text.unwrap_or(""))),
```

Comment:
Can you think of a way to refactor this 4-line initializer? Looks like a lot to unpack, maybe we could create the `Input` and modify it before calling `Self {}`?

Reply:
Something like that is enough?
https://github.com/mikaelmello/inquire/pull/34/commits/25ffaa4843e3f2f723df13b70fdeab94fddbc166#diff-2265875200c3d6bf45336f1bd31a5c62580f1511c5e846de5973611255a508cbR274-R280